config/locales/client.ar.yml

```diff
@@ -4818,7 +4818,7 @@ ar:
             badges:
               content: "الشارات"
               title: "كل الشارات الممكن منحها لك"
-            everything:
+            topics:
               content: "كل شيء"
               title: "كل الموضوعات"
```

Comment:
I think we should delete the translations in the non-English locales instead of renaming the keys because the translation for "Topics" isn't the same as "Everything".